assets/shared/images/NOTE_assets.xml

```diff
@@ -6,7 +6,6 @@
 	<SubTexture name="arrowLEFT0000" x="310" y="235" width="153" height="157"/>
 	<SubTexture name="arrowRIGHT0000" x="157" y="235" width="153" height="157"/>
 	<SubTexture name="arrowUP0000" x="784" y="232" width="157" height="153"/>
-	<SubTexture name="blue0000" x="1850" y="154" width="157" height="154"/>
```

Comment:
Unrelated to the PR's purpose, please revert this.

Reply:
I don't know how to do that tbh.

Reply:
just realized i need github desktop lol.

Reply:
No... you just need Git.